docker/appserver/WebSphere/src/scripts/test-with-mssql.bat

```diff
@@ -0,0 +1,4 @@
+cd %~dp0/../..
+docker container stop iaf-test-as-websphere-with-mssql
+docker container rm iaf-test-as-websphere-with-mssql
+docker run --publish 9443:9443 --publish 9043:9043  -e larva.adapter.execute= -e jdbc.dbms.default=mssql --name iaf-test-as-websphere-with-mssql iaf-test-as-websphere:7.6-SNAPSHOT
```

Comment:
je gebruikt nu wel overal de versie nummer. kan die niet ergens dynamisch uitgehaald worden? je kan in een batch script een ander batch script aanroepen oid? 
Al leest hij maar 1 bestand uit, is al beter dan bij elke FF versie 15 bestanden aanpassen..